R/arrange_regression.R

```diff
@@ -68,8 +69,9 @@ arrange_regression <- function(x, est_name, standardized, conf.int, ...) {
     , statistic = test_statistic
     , p.value   = "$p$"
   )
+  if(!is.null(regression_table$df)) variable_label(regression_table$df) <- "$\\mathit{df}$"
 
-  class(regression_table) <- c("apa_regression_table", class(regression_table))
+  class(regression_table) <- c("apa_results_table", class(regression_table))
```

Comment:
Just out of curiosity, can you remind me, why we (I?) defined a separate class here? 😅